README.md

```diff
@@ -2,6 +2,23 @@
 
 > Our guidelines for how we do Scrum
 
+## Increments
+
+Once a potentially shippable increment is ready as the result of a finished sprint, the associated product should have its version incremented according to [Semantic Versioning](http://semver.org/). 
+
+A [GitHub release](https://help.github.com/articles/creating-releases/) is then to be created with a high-level description of the changes made during the sprint and a [changelog](http://keepachangelog.com/) reflecting the exact items finished. Both the tag version and release title must be the product version prefixed by a single _v_ (eg. `v1.0.0`). The following template is to be used for the release description:
+
+```md
+This release implements some new feature and fixes various bugs.
+
+### Changes
+
+- #1 Implemented feature xyz
+- #2 Fixed bug caused by xyz
+
```

Comment:
Should we separate PRs and issues into two subsections, or would that be excessive?
```
### Changes

## Issues
#1 Implemented feature xyz

## Pull requests
#2 Fixed bug caused by xyz
```